apartment_application_service/settings.py

```diff
@@ -176,8 +176,35 @@
 LOGGING = {
     "version": 1,
     "disable_existing_loggers": False,
-    "handlers": {"console": {"class": "logging.StreamHandler"}},
-    "loggers": {"django": {"handlers": ["console"], "level": env.str("LOG_LEVEL")}},
+    "formatters": {
+        "standard": {
+            "format": "%(asctime)s p%(process)d %(name)s %(levelname)s: %(message)s",
+            "datefmt": "%Y-%m-%dT%H:%M:%S.%03dZ",
```

Comment:
There's a small bug here, the date will be printed like this:

    2021-05-20T10:58:06.3dZ p3252 django.server INFO: "GET /v1/ HTTP/1.1" 200 10314

Note the `.3dZ` after the seconds. Apparently the problem here is that there is actually no way to put the milliseconds in the `datefmt` (although you could use `%(msecs)03d` in the `format` string).

Another issue is that the `Z` at the end of `datefmt` means the timestamp is UTC, however it actually uses local time. So I think we don't really want the `Z` there.

I think simplest solution to both of these problems would be to simply remove `datefmt`. Then the output would look like this:

    2021-05-20 11:03:57,441 p3284 django.server INFO: "GET /v1/ HTTP/1.1" 200 10314

What do you think?